[Security Solution] Implement prebuilt rules import test plan

maximpn opened the pull request:
**Addresses:** https://github.com/elastic/kibana/issues/202079

## Summary

This PR implement prebuilt rules import [test plan](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_import.md).

Prior existing tests were superseded by the new ones.

## Discovered issues

It turned out that the current implementation behaves slightly differently than described the test plan. Failed test scenarios are skipped and the following tickets have been created for tracking

- https://github.com/elastic/kibana/issues/223099https://github.com/elastic/kibana/issues/223099
- https://github.com/elastic/kibana/issues/223280
- https://github.com/elastic/kibana/issues/223253

## Extra changes

Some diff like in `alert_threat_enrichments.cy.ts` has been caused by disabling errors swallowing upon rule creation. It's important to have tests properly preparing the testing environment to reduce chances of test dependencies and flakiness. In particular some prebuilt rules import test scenarios require custom rules to be created. Swallowing rule creation errors reduces tests coverage. 

## Flaky test runner

- ✅  [`detection_engine_security` package related prebuilt rules import integration tests](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8365) (100 iterations)
- ✅ [prebuilt rules import ECH integration tests](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8366) (100 iterations)
- ✅ [prebuilt rules import Serverless integration tests](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8367) (100 iterations)



<!--ONMERGE {"backportTargets":["8.19"]} ONMERGE-->

<!--ONMERGE {"backportTargets":["8.19","9.0","9.1"]} ONMERGE-->

<!--ONMERGE {"backportTargets":["8.18","8.19","9.0","9.1"]} ONMERGE-->

maximpn committed:
add import single prebuilt rule tests

maximpn committed:
add a function to prevent real package installation

maximpn committed:
add prebuilt rule import tests

maximpn committed:
move common prebuilt rules import tests to the common folder

maximpn committed:
add package dependent prebuilt rules import tests

maximpn committed:
remove duplicated integration tests

maximpn committed:
fix return type

maximpn committed:
update test plan status

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

maximpn commented:
/ci

maximpn commented:
/ci

kibanamachine commented:
## Flaky Test Runner Stats
### 🎉 All tests passed! - [kibana-flaky-test-suite-runner#8365](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8365)
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/configs/ess_air_gapped.config.ts: 100/100 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F223421%2Fhead)

kibanamachine commented:
## Flaky Test Runner Stats
### 🎉 All tests passed! - [kibana-flaky-test-suite-runner#8367](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8367)
[✅] x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/configs/serverless_essentials_tier.config.ts: 100/100 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F223421%2Fhead)

elasticmachine commented:
Pinging @elastic/security-detections-response (Team:Detections and Resp)

elasticmachine commented:
Pinging @elastic/security-solution (Team: SecuritySolution)

elasticmachine commented:
Pinging @elastic/security-detection-rule-management (Team:Detection Rule Management)

maximpn commented:
/ci

jbudz reviewed (approved):
`.buildkite/ftr_security_configs.yml` LGTM

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309834)
* Commit: 650243a9c2629b6a8c874d4aca553800a254e3eb

### Failed CI Steps
* [Jest Tests #15](https://buildkite.com/elastic/kibana-pull-request/builds/309834#019782a1-f35c-4bf4-95d0-d423feadc1a1)
* [FTR Configs #110](https://buildkite.com/elastic/kibana-pull-request/builds/309834#019782a1-f3bd-4024-9833-e84ad4b22cfd)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/309834#019782a1-f3bd-4024-9833-e84ad4b22cfd) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/309834/jobs/019782a1-f3bd-4024-9833-e84ad4b22cfd/artifacts/019782b2-a437-4715-9467-a52b4190d5bd) FTR Configs #<span></span>110 / alerting api integration security and spaces enabled - Group 2 Connectors ServiceNow SIR ServiceNow SIR - Executor Execution getChoices should get choices
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/309834#019782a1-f35c-4bf4-95d0-d423feadc1a1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/309834/jobs/019782a1-f35c-4bf4-95d0-d423feadc1a1/artifacts/019782c6-703a-4cf0-8f86-21f7ae9e7d5a) Jest Tests #<span></span>15 / When using the `useGetFileInfo()` hook should call the correct api with expected value

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :yellow_heart: [Build #307677](https://buildkite.com/elastic/kibana-pull-request/builds/307677) was flaky 4e550407eb6ac1618049be5f8f508f136a040482
* :broken_heart: [Build #307467](https://buildkite.com/elastic/kibana-pull-request/builds/307467) failed bd99262f012c38eb60749b6ac5d472312262708a

cc @maximpn

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309834","number":309834,"commit":"650243a9c2629b6a8c874d4aca553800a254e3eb"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307677","number":307677,"commit":"4e550407eb6ac1618049be5f8f508f136a040482"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307467","number":307467,"commit":"bd99262f012c38eb60749b6ac5d472312262708a"}],"number":309834}
buildkite-pr-comment-->

dplumlee reviewed (commented):
Changes look really uniform and structured @maximpn 👍, I left a some comments/questions and I think there are a couple test cases that have errors but then I will approve

maximpn committed:
fix incorrect import payload

maximpn commented on x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/import/import_single_prebuilt_rule.ts:
```diff
@@ -0,0 +1,711 @@
+            immutable: true,
+            rule_source: {
+              type: 'external',
+              is_customized: false,
```
A good catch. It's a misprint and definitely it should be `true`. Though `rule_source.is_customized` value in the import payload doesn't impact the imported value since we recalculate it anyway. I set it for consistency or if we start take it into account so we have the coverage.

maximpn committed:
fix prebuilt rule import with different versions

maximpn committed:
move import rules utils to the rule management utils folder

maximpn commented on x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/import/utils.ts:
```diff
@@ -0,0 +1,75 @@
+  overwrite: boolean;
+}
+
+export async function importRules({
```
Yes. It makes sense. I've moved import rules utilities to `x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules` since these aren't prebuilt rules specific.

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310806)
* Commit: a6b879e87a1da2a7bbbc3314a08beef5863ba514

### Failed CI Steps
* [FTR Configs #133](https://buildkite.com/elastic/kibana-pull-request/builds/310806#01978cf4-b1bf-42a1-bae5-3c94bec35405)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/310806#01978cf4-b1bf-42a1-bae5-3c94bec35405) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/310806/jobs/01978cf4-b1bf-42a1-bae5-3c94bec35405/artifacts/01978d10-98aa-4a3e-9fae-a97b37b13e57) FTR Configs #<span></span>133 / Actions and Triggers app Connectors Opsgenie connector page should reset connector when canceling an edit

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :yellow_heart: [Build #309834](https://buildkite.com/elastic/kibana-pull-request/builds/309834) was flaky 650243a9c2629b6a8c874d4aca553800a254e3eb
* :yellow_heart: [Build #307677](https://buildkite.com/elastic/kibana-pull-request/builds/307677) was flaky 4e550407eb6ac1618049be5f8f508f136a040482
* :broken_heart: [Build #307467](https://buildkite.com/elastic/kibana-pull-request/builds/307467) failed bd99262f012c38eb60749b6ac5d472312262708a

cc @maximpn

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310806","number":310806,"commit":"a6b879e87a1da2a7bbbc3314a08beef5863ba514"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309834","number":309834,"commit":"650243a9c2629b6a8c874d4aca553800a254e3eb"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307677","number":307677,"commit":"4e550407eb6ac1618049be5f8f508f136a040482"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307467","number":307467,"commit":"bd99262f012c38eb60749b6ac5d472312262708a"}],"number":310806}
buildkite-pr-comment-->

maximpn committed:
add prebuilt rules import e2e tests

maximpn committed:
add extra missing fields tests

maximpn commented:
@dplumlee thanks for reviewing the test plan implementation 🙏  I've addressed your comments and added a couple of Cypress tests. Could you have a look?

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310893)
* Commit: 861c468285b1c38e55f5047e2220061847f1a2ac

### Failed CI Steps
* [Detection Engine - Security Solution Cypress Tests #4](https://buildkite.com/elastic/kibana-pull-request/builds/310893#01978d79-6907-4fc5-a297-3e2839dde441)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/310893#01978d79-6907-4fc5-a297-3e2839dde441) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/310893/jobs/01978d79-6907-4fc5-a297-3e2839dde441/artifacts/01978da8-4c0e-4f3d-bc38-79f9aaba1a4a) Detection Engine - Security Solution Cypress Tests #<span></span>4 / Threat Match Enrichment "before each" hook for "Displays threat indicator details on the threat intel tab" "before each" hook for "Displays threat indicator details on the threat intel tab"

### History
* :yellow_heart: [Build #310806](https://buildkite.com/elastic/kibana-pull-request/builds/310806) was flaky a6b879e87a1da2a7bbbc3314a08beef5863ba514
* :yellow_heart: [Build #309834](https://buildkite.com/elastic/kibana-pull-request/builds/309834) was flaky 650243a9c2629b6a8c874d4aca553800a254e3eb
* :yellow_heart: [Build #307677](https://buildkite.com/elastic/kibana-pull-request/builds/307677) was flaky 4e550407eb6ac1618049be5f8f508f136a040482
* :broken_heart: [Build #307467](https://buildkite.com/elastic/kibana-pull-request/builds/307467) failed bd99262f012c38eb60749b6ac5d472312262708a

cc @maximpn

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310893","number":310893,"commit":"861c468285b1c38e55f5047e2220061847f1a2ac"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310806","number":310806,"commit":"a6b879e87a1da2a7bbbc3314a08beef5863ba514"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309834","number":309834,"commit":"650243a9c2629b6a8c874d4aca553800a254e3eb"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307677","number":307677,"commit":"4e550407eb6ac1618049be5f8f508f136a040482"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307467","number":307467,"commit":"bd99262f012c38eb60749b6ac5d472312262708a"}],"number":310893}
buildkite-pr-comment-->

maximpn committed:
extend descriptions

maximpn committed:
restrict cache invalidation to ECH only

maximpn committed:
fix failing Cypress tests

dplumlee reviewed (approved):
Thanks for addressing my comments @maximpn, new cypress tests look good as well 👍

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311151)
* Commit: c90f6178cfb38a0ba64e65afedeec89a11e3bb70

### Failed CI Steps
* [FTR Configs #94](https://buildkite.com/elastic/kibana-pull-request/builds/311151#01979245-4564-4b75-ac43-4af4fd6a9b24)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/311151#01979245-4564-4b75-ac43-4af4fd6a9b24) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/311151/jobs/01979245-4564-4b75-ac43-4af4fd6a9b24/artifacts/0197926b-708c-4e15-9fb7-aea04736cb27) FTR Configs #<span></span>94 / ESQL execution logic API @<span></span>ess @<span></span>serverless ES|QL rule type shard failures should handle shard failures and include warning in logs for query that is not aggregating

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #311096](https://buildkite.com/elastic/kibana-pull-request/builds/311096) failed 991a6baed27875501fd67dd2caf2078a018a4d3a
* :broken_heart: [Build #310893](https://buildkite.com/elastic/kibana-pull-request/builds/310893) failed 861c468285b1c38e55f5047e2220061847f1a2ac
* :yellow_heart: [Build #310806](https://buildkite.com/elastic/kibana-pull-request/builds/310806) was flaky a6b879e87a1da2a7bbbc3314a08beef5863ba514
* :yellow_heart: [Build #309834](https://buildkite.com/elastic/kibana-pull-request/builds/309834) was flaky 650243a9c2629b6a8c874d4aca553800a254e3eb
* :yellow_heart: [Build #307677](https://buildkite.com/elastic/kibana-pull-request/builds/307677) was flaky 4e550407eb6ac1618049be5f8f508f136a040482

cc @maximpn

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311151","number":311151,"commit":"c90f6178cfb38a0ba64e65afedeec89a11e3bb70"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311096","number":311096,"commit":"991a6baed27875501fd67dd2caf2078a018a4d3a"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310893","number":310893,"commit":"861c468285b1c38e55f5047e2220061847f1a2ac"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310806","number":310806,"commit":"a6b879e87a1da2a7bbbc3314a08beef5863ba514"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309834","number":309834,"commit":"650243a9c2629b6a8c874d4aca553800a254e3eb"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307677","number":307677,"commit":"4e550407eb6ac1618049be5f8f508f136a040482"}],"number":311151}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311168)
* Commit: 3e281171821a0f5f9fee5131f6f846afcbc0bc41

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :yellow_heart: [Build #311151](https://buildkite.com/elastic/kibana-pull-request/builds/311151) was flaky c90f6178cfb38a0ba64e65afedeec89a11e3bb70
* :broken_heart: [Build #311096](https://buildkite.com/elastic/kibana-pull-request/builds/311096) failed 991a6baed27875501fd67dd2caf2078a018a4d3a
* :broken_heart: [Build #310893](https://buildkite.com/elastic/kibana-pull-request/builds/310893) failed 861c468285b1c38e55f5047e2220061847f1a2ac
* :yellow_heart: [Build #310806](https://buildkite.com/elastic/kibana-pull-request/builds/310806) was flaky a6b879e87a1da2a7bbbc3314a08beef5863ba514
* :yellow_heart: [Build #309834](https://buildkite.com/elastic/kibana-pull-request/builds/309834) was flaky 650243a9c2629b6a8c874d4aca553800a254e3eb
* :yellow_heart: [Build #307677](https://buildkite.com/elastic/kibana-pull-request/builds/307677) was flaky 4e550407eb6ac1618049be5f8f508f136a040482

cc @maximpn

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311168","number":311168,"commit":"3e281171821a0f5f9fee5131f6f846afcbc0bc41"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311151","number":311151,"commit":"c90f6178cfb38a0ba64e65afedeec89a11e3bb70"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311096","number":311096,"commit":"991a6baed27875501fd67dd2caf2078a018a4d3a"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310893","number":310893,"commit":"861c468285b1c38e55f5047e2220061847f1a2ac"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310806","number":310806,"commit":"a6b879e87a1da2a7bbbc3314a08beef5863ba514"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309834","number":309834,"commit":"650243a9c2629b6a8c874d4aca553800a254e3eb"}],"number":311168}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311169)
* Commit: b92118d215603d405177f409f696c8397fd10b51

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :green_heart: [Build #311168](https://buildkite.com/elastic/kibana-pull-request/builds/311168) succeeded 3e281171821a0f5f9fee5131f6f846afcbc0bc41
* :yellow_heart: [Build #311151](https://buildkite.com/elastic/kibana-pull-request/builds/311151) was flaky c90f6178cfb38a0ba64e65afedeec89a11e3bb70
* :broken_heart: [Build #311096](https://buildkite.com/elastic/kibana-pull-request/builds/311096) failed 991a6baed27875501fd67dd2caf2078a018a4d3a
* :broken_heart: [Build #310893](https://buildkite.com/elastic/kibana-pull-request/builds/310893) failed 861c468285b1c38e55f5047e2220061847f1a2ac
* :yellow_heart: [Build #310806](https://buildkite.com/elastic/kibana-pull-request/builds/310806) was flaky a6b879e87a1da2a7bbbc3314a08beef5863ba514
* :yellow_heart: [Build #309834](https://buildkite.com/elastic/kibana-pull-request/builds/309834) was flaky 650243a9c2629b6a8c874d4aca553800a254e3eb

cc @maximpn

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311169","number":311169,"commit":"b92118d215603d405177f409f696c8397fd10b51"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311168","number":311168,"commit":"3e281171821a0f5f9fee5131f6f846afcbc0bc41"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311151","number":311151,"commit":"c90f6178cfb38a0ba64e65afedeec89a11e3bb70"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311096","number":311096,"commit":"991a6baed27875501fd67dd2caf2078a018a4d3a"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310893","number":310893,"commit":"861c468285b1c38e55f5047e2220061847f1a2ac"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310806","number":310806,"commit":"a6b879e87a1da2a7bbbc3314a08beef5863ba514"}],"number":311169}
buildkite-pr-comment-->

vgomez-el reviewed (approved):
LGTM!

maximpn committed:
Merge branch 'main' into address-import-prebuilt-rules-test-plan

nkhristinin reviewed (approved):
code review only for DE, LGTM!

kibanamachine commented:
Starting backport for target branches: 8.19

https://github.com/elastic/kibana/actions/runs/15850342478

maximpn commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226448">](https://github.com/elastic/kibana/pull/226448)|
|✅|8.18|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226449">](https://github.com/elastic/kibana/pull/226449)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 10.0.1 -->

kibanamachine commented:
Starting backport for target branches: 8.19, 9.0, 9.1

https://github.com/elastic/kibana/actions/runs/16061804144

kibanamachine commented:
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|8.19|Backport failed because of merge conflicts|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226507">](https://github.com/elastic/kibana/pull/226507)|
|❌|9.1|Backport failed because of merge conflicts|

Note: Successful backport PRs will be merged automatically after passing CI.

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 223421
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

kibanamachine commented:
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|8.19|Backport failed because of merge conflicts|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226507">](https://github.com/elastic/kibana/pull/226507)|
|❌|9.1|Backport failed because of merge conflicts|

Note: Successful backport PRs will be merged automatically after passing CI.

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 223421
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

kibanamachine commented:
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.18|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226508">](https://github.com/elastic/kibana/pull/226508)|
|❌|8.19|Backport failed because of merge conflicts|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226507">](https://github.com/elastic/kibana/pull/226507)|
|❌|9.1|Backport failed because of merge conflicts|

Note: Successful backport PRs will be merged automatically after passing CI.

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 223421
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

maximpn commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|9.1|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226542">](https://github.com/elastic/kibana/pull/226542)|
|✅|8.19|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226543">](https://github.com/elastic/kibana/pull/226543)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 10.0.1 -->
